### PR TITLE
updates re: custom kab. crd, step order

### PR DESCRIPTION
**Pull request by jantley-ibm**

### CODE REVIEW

File: ref/general/collection-building.adoc [hunk 1]
```diff
@@ -15,6 +15,17 @@ cd collections
 +
 Once this has been done, you will have your own copy of the collections repository in your own git org/repo. Follow any normal development processes you have for using git (i.e., creating branches, etc.).
 
+. Create and organizational-level teams in Git that should administer the cloned collection, and add members to those teams.
```
Comment (davco01a) [inline ref/general/collection-building.adoc]: Create organizational-level team(s)......that should administer the cloned collection(s)

File: ref/general/collection-building.adoc [hunk 2]
```diff
@@ -15,6 +15,17 @@ cd collections
...
+
+. Set up a Kabanero Custom Resource Definition to target the new collections
+* Specify the repository URL in the Kabanero document (object) to point to cloned, customized index.yaml for the master collection.
+You can either alter the Kabanero document that already exists, or delete it and create a new one.
+* In the `kabanero crd`, specify:
+  git `group or organization` in the `organization` field
```
Comment (davco01a) [inline ref/general/collection-building.adoc]: I think this may be just `git organization` not sure where we got started with group

Comment (davco01a) [reply ref/general/collection-building.adoc]: and we may be switching to multiple orgs too

File: ref/general/collection-building.adoc [hunk 3]
```diff
@@ -15,6 +15,17 @@ cd collections
 +
 Once this has been done, you will have your own copy of the collections repository in your own git org/repo. Follow any normal development processes you have for using git (i.e., creating branches, etc.).
 
+. Create and organizational-level teams in Git that should administer the cloned collection, and add members to those teams.
+
+. Set up a Kabanero Custom Resource Definition to target the new collections
+* Specify the repository URL in the Kabanero document (object) to point to cloned, customized index.yaml for the master collection.
+You can either alter the Kabanero document that already exists, or delete it and create a new one.
+* In the `kabanero crd`, specify:
```
Comment (kaczyns) [inline ref/general/collection-building.adoc]: Technically, the `kabanero crd` is the template that describes what a Kabanero instance (what we've been calling the Kabanero document) looks like.  So you wouldn't specify this "in" the "kabanero crd", you'd specify it in the kabanero instance (or kabanero cr instance).

File: ref/general/collection-building.adoc [hunk 4]
```diff
@@ -15,6 +15,17 @@ cd collections
...
+  git `group or organization` in the `organization` field
+  git `team name(s)` in the `teams` field
+  the `api github url` in the `apiurl` field. ( env variable: github.api.url )
+  the kabanero `namespace(s)` in the `namespaces` field
```
Comment (kaczyns) [inline ref/general/collection-building.adoc]: The `namespaces` field does not exist yet.  I don't think we should be adding it to the documentation yet.

File: ref/general/collection-building.adoc (hunk 1 above)
Comment (jantley-ibm) [reply ref/general/collection-building.adoc]: yes, thanks

File: ref/general/collection-building.adoc (hunk 3 above)
Comment (jantley-ibm) [reply ref/general/collection-building.adoc]: Do you prefer "instance" or "document"?  Is it one or the other in all places?  Should I also change "Custom Resource Definition" in line 20 to "document" (or "instance")?

Comment (kaczyns) [reply ref/general/collection-building.adoc]: I was reading thru the kubernetes doc to see how they refer to them.  It looks like they say "custom objects" but I've never heard anyone refer to them that way.  I guess I prefer `Kabanero instance` and we should be consistent with how we refer to it.  Technically it should probably be `Kabanero CR instance` where CR stands for Custom Resource (and CRD stands for Custom Resource Definition).

I think just saying `Kabanero instance` will get the point across.  But if others disagree it won't take much to change my opinion.

Comment (kaczyns) [reply ref/general/collection-building.adoc]: Oh, line 20, should say "Set up a Kabanero instance to target the new collections."
Then line 21 "Specify the repository URL in the Kabanero instance to point to cloned, customized index.yaml for the master collection.  The URL is set in the `Spec.Collections.Repositories` section of the Kabanero instance."
Then line 22 I think just replace document with instance.

**Comment by jantley-ibm** [comment]
@davco01a @kaczyns updates made, could one of you approve and merge the PR?  Thanks!

File: ref/general/collection-building.adoc [hunk 5]
```diff
@@ -15,16 +15,15 @@ cd collections
...
 
-. Set up a Kabanero Custom Resource Definition to target the new collections
-* Specify the repository URL in the Kabanero document (object) to point to cloned, customized index.yaml for the master collection.
-You can either alter the Kabanero document that already exists, or delete it and create a new one.
-* In the `kabanero crd`, specify:
-  git `group or organization` in the `organization` field
+. Set up a Kabanero instance to target the new collections
+* Specify the repository URL in the Kabanero document (object) to point to cloned, customized `index.yaml` for the master collection.  The URL setting is in the `Spec.Collections.Repositories` section of the Kabanero instance.
+You can either alter the Kabanero instance that already exists, or delete it and create a new one.
+* In the Kabanero document, specify:
```
Comment (kaczyns) [inline ref/general/collection-building.adoc]: This should also be Kabanero instance, not Kabanero document.

File: ref/general/collection-building.adoc [hunk 6]
```diff
@@ -15,16 +15,15 @@ cd collections
...
+  git `organizations` in the `organizations` field
```
Comment (kaczyns) [inline ref/general/collection-building.adoc]: The name of the field today is 'organization', and we only support a single one today.  I think it's premature to imply we support multiple organizations today, even if we intend to shortly.

Comment (kaczyns) [reply ref/general/collection-building.adoc]: Also online 26, I'm not sure why it references the name of the environment variable - that isn't going to mean anything to the user.  @davco01a @brutif do you know why that is there?

Comment (brutif) [reply ref/general/collection-building.adoc]: @kaczyns  no, but I do wonder how the orgs and teams get merged together.  Security uses the "fully qualified" team name of team_name@org.  so if we have orgs a and b, and teams t1 and t2, what fq names are passed to security?  t1@a, t1@b, both?

Comment (kaczyns) [reply ref/general/collection-building.adoc]: There is only one org supported today, so the Kab operator merges them together with the @ sign team1@org,team2@org,team3@org

**Comment by jantley-ibm** [comment]
No problem @kaczyns , updates made

**Comment by marikaj123** [comment]
@alohr51 - This is ready fro publish.

**Pull request merged**